.circleci/config.yml

```diff
@@ -98,6 +98,9 @@ jobs:
       - run:
           name: Smart Contract Tests
           command: scripts/tests.sh smart-contracts
+      - run: 
+          name: Codechecks
+          command: cd smart-contracts && npm i @codechecks/client && npx codechecks
```

Comment:
I am not a huge fan of `npx` which will install the package?
Could we instead add `@codechecks/client` as a dev dependency and a corresponding script `codechecks` ( and put `npm run codechecks` in here)?

Reply:
hmm I tried something like that in a previous commit, it didn't work.. i think it's due to the `--production` flag.  Can we try turning that off?

npx will not install the package since it's scoped by name.  that's why i have the `npm i` as well.

Reply:
hmm nm maybe it did work, i can revert and try that again